corehq/apps/app_manager/views.py

```diff
@@ -1716,6 +1717,8 @@ def rename_action_case_type(mod):
             app.enable_module_filtering and
             should_edit('module_filter')):
         module['module_filter'] = request.POST.get('module_filter')
+        if '#user' in module['module_filter'] and not is_usercase_in_use(domain):
```

Comment:
True. I put it in because `enable_usercase` checks `is_usercase_in_use` inside a CriticalSection block, which I figured would be slower. But I'm sure it's not significant and taking out the check here makes it simpler. :+1: